### ISSUE TITLE
[Programmatic Access – Make Code - Print] Aria-expanded property is not defined for the  control under 'Print' blade.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3773]
User Experience:
If, the aria-expanded property is not defined for the control, then the user who depends then the user who depends upon Screen Reader for navigation might gets affected hence failed to gets the proper information of the control.

Note: User credentials should NOT be included in the bug.

Pre-Requisite:
Open F12 Developer tool.

Repro Steps:

1. Launch application URL
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Print" option and select it.
7. Navigate through all the controls appearing under "Print Code" page
8. Verify the issue.

Actual Result:
Aria-expanded property is not defined for the  control under 'print' blade.

Expected Result:
Aria-expanded property should be defined for the  control under 'print' blade. So, that the Screen Reader user does able to gets the proper information of the button.

Note:
Same issue exists on Firefox + NVDA & Chrome + JAWS.

WCAG Reference:
[WCAG 4.1.2 – Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html%7D)

Test Environments:
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0), Firefox(75.0), Chrome
Screen Reader: Narrator, JAWS, NVDA

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11yMAS; #A11ySev2; #HCL; #HCL-MakeCode-New; #WCAG4.1.2; #Win10-Edge(Chromium)

![WCAG4 1 2](https://user-images.githubusercontent.com/72871089/100340080-ab7a8a00-3000-11eb-8860-0c6d584f0956.png)

**Comment by jwunderl** [comment]
This is the browsers print dialog, we do not have control over this. Closing.
